lib/tasks/user.rake

```diff
@@ -10,4 +10,16 @@ namespace :user do
     )
     user.roles.create! role_type: args[:role]
   end
+
+  task :deactivate, [:email] => [:environment] do |t, args|
```

Comment:
Since this just takes 1 argument, we can probably simplify this to `rake user:deactivate test@test.com` and also add in some error checking for missing users

Reply:
i've added some error checking but kept arg format as this aligns with format of other tasks we have